sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -1086,13 +1088,13 @@ def set_http_headers(self, content_settings,  # type: ContentSettings
             The file system attributes for files and directories.
             If not set, indicates preservation of existing values.
             Here is an example for when the var type is str: 'Temporary|Archive'
-        :type file_attributes: str or :class:`~azure.storage.fileshare.NTFSAttributes`
+        :type file_attributes: str or NTFSAttributes
         :param file_creation_time: Creation time for the file
             Default value: Preserve.
-        :type file_creation_time: str or ~datetime.datetime
+        :type file_creation_time: str or datetime
```

Comment:
This also needs to be reverted. I can show you a trick (that Jacob showed me) that can help demystify this process instead of it being sort of black-box-y

But for example, this is how your docs is rendering this now:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/59724171-954c-4325-a8c1-6ea6d1b9d99b)
Notice how there is no hyperlink for `datetime`

Here is what the docs rendered leaving `~datetime.datetime` intact:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/6f932d79-0637-4bd7-af32-1f0e13ca5153)

We can discuss this in our meeting tomorrow, so no worries.